_rules/html-page-title-descriptive-c4a8a4.md

```diff
@@ -159,6 +159,22 @@ This `<title>` element, the first of two, does not describe the content of the d
 </html>
 ```
 
+#### Failed Example 3
+
+This page has a generic [document title][]. The title contains the website name, but does not describe the page.
+
+```html
+<html lang="en">
+	<head>
+		<title>University of Arkham</title>
+	</head>
+	<body>
+		<h1>Search results for "accessibility"</h1>
```

Comment:
Maybe something like
```suggestion
		<h1>Search results for "accessibility" at the University of Arkham</h1>
```
to make it clear how the title is "generic".
(I guess the idea here is that all pages of the university have the title "University of Arkham", which is non-descriptive of the specific page, so trying to convey in the page this idea of "this is a page of the University of Arkham, so the title is not 100% stupid, but still not enough")